README.md

```diff
@@ -19,7 +19,7 @@ Documenting Bluetooth services and characteristics of the Unu Scooter Pro
 | 9a590070-6e67-5d0d-aab9-ad9126b66f91 | 9a590072-6e67-5d0d-aab9-ad9126b66f91 | "not-charging"<br>"charging"<br>(Probably more) | CB battery charge status |
 | 9a590100-6e67-5d0d-aab9-ad9126b66f91 |  |  | TBD. Could very well be related to GPS |
 |                                      | 9a590101-6e67-5d0d-aab9-ad9126b66f91 | "cbb"<br>"aux"<br>(Probably more) | Battery type. TBD |
-| TBD                                  | 9a5900a1-6e67-5d0d-aab9-ad9126b66f91 | "hibernating"<br>"booting"<br>"running"<br>(Probably more) | TBD |
+| TBD                                  | 9a5900a1-6e67-5d0d-aab9-ad9126b66f91 | "hibernating"<br>"booting"<br>"running"<br>"suspending"<br>"hibernating-imminent"<br>(Probably more) | TBD |
```

Comment:
I think a combination of `9a590020 stand-by` and `9a5900a1 hibernating-imminent` should reflect the hibernation state of the scooter which should be displayed in the app